src/Behavior/NodeException.php

```diff
@@ -29,13 +30,20 @@ public static function create(): self
      */
     protected $node;
 
-    public function withNode(?DOMNode $node): self
+    /**
+     * @param \DOMNode|null $node
+     * @return $this
+     */
+    public function withNode($node)
```

Comment:
Parameter typing for classes worked since PHP5, `DOMNode $node = null` in this case would work